index.html

```diff
@@ -227,8 +215,29 @@ <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
           </ul>
       </div>
     </section>
+    <section class="contact">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+    <form action="https://formspree.io/f/xgebvavj" method="POST" class="contact-form">
+      <input type="text" class="contact-input" placeholder="Full name" maxlength="30" required>
+      <input type="text" class="contact-input" placeholder="Email address" required>
+      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500">Enter text here</textarea>
+      <button class="contact-btn" type="button">Get in touch</button>
```

Comment:
This is a nice button 😉 but if you want it to work with the `form` you need to use `type="submit"` instead of `type="button"`, you may be wondering why 🤔 Here's why:
  - The `<button type="submit">` specifies a submit button. Clicking a submit button sends form data to a form handler on the server.
  
 Feel free to reply to that thread if you have any questions 💯